Makefile

```diff
@@ -80,6 +66,10 @@ ifeq ($(SOURCE_OS),darwin)
 	SUBST = $(HOMEBREW_PREFIX)/Cellar/x264/r3108/lib/libx264.a
 endif
 CGO_LDFLAGS = $(subst -lx264, $(SUBST),$(shell PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --libs $(FFMPEG_LIBS))) 
+# if windows add  -lpthread
+ifeq ($(TARGET_OS),windows)
+	CGO_LDFLAGS += -lpthread -static -static-libgcc -static-libstdc++
```

Comment:
why do we need `-static-libstdc++`?

Reply:
For some reason the binary has a dangling DLL (`libstdc++-6.dll`) without this flag.

```
x86_64-w64-mingw32-objdump -p viamrtsp.exe | grep 'DLL Name'
        DLL Name: bcrypt.dll
        DLL Name: KERNEL32.dll
        DLL Name: msvcrt.dll
        DLL Name: ole32.dll
        DLL Name: QUARTZ.dll
        DLL Name: USER32.dll
        DLL Name: libgcc_s_seh-1.dll
        DLL Name: libstdc++-6.dll
        ~``